nautobot/extras/api/customfields.py

```diff
@@ -110,3 +110,9 @@ def get_field_names(self, declared_fields, info):
         self.extend_field_names(fields, "custom_fields")
         self.extend_field_names(fields, "computed_fields", opt_in_only=True)
         return fields
+
+
+# TODO: remove in 2.2
```

Comment:
IMO we should go ahead and backlog these and attach the issue(s) to the 2.2 milestone.

Reply:
Good call. Opened #2850.